WordPress/lint-baseline.xml

```diff
@@ -1,28 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <issues format="4" by="lint 3.1.3">
 
-    <issue
```

Comment:
@jtreanor Could you explain the reasons for dropping some these entries from `lint-baseline.xml`? On the face of it it's not clear why they belong in this PR, but I'm not too familiar with their background.

Reply:
Lint gave a bunch of warnings about issues in `lint-baseline.xml` (such as the Gradle upgrade warnings) were no longer matching lint results. Some of these warnings may not have been new but I removed the ones that I saw.

Would you prefer if I didn't remove the rules that are not directly related to this PR?

Reply:
@jtreanor looks like they were present in `develop` as well - I don't see any harm in dropping them, so let's just leave those changes in 👍 Just wanted to be clear on any background there might have been.